src/util/api/index.js

```diff
@@ -28,6 +28,12 @@ export const getCivicLikerTrialEventByIdAPI = id =>
 export const getCivicLikerJoinTrialEventByIdAPI = id =>
   `/api/civic/trial/events/${id}/join`;
 
+export const getImageResizeAPI = (url, { width } = {}) =>
+  `https://us-central1-civic-liker${
+    IS_TESTNET ? '-develop' : ''
+  }.cloudfunctions.net/thumbnail/?url=${encodeURIComponent(url)}${
```

Comment:
Should pass host in env var?

Reply:
probably want to use it in dev env too to avoid unwanted diff in behaviour...
not sure

Reply:
can others use that API publicly? If so, is that a concern?

Reply:
probably can as in we didnt put any real checking here except origin header, and this url is very visible under browser network inspector